babeld/babel_interface.c

```diff
@@ -606,6 +688,24 @@ DEFUN (babel_set_channel_noninterfering,
     return CMD_SUCCESS;
 }
 
+DEFPY (no_babel_set_channel,
+       no_babel_set_channel_cmd,
+       "no babel channel [CHANNEL]",
+       NO_STR
+       "Babel interface commands\n"
+       "Channel number for diversity routing\n"
+       "Channel\n")
+{
+    VTY_DECLVAR_CONTEXT(interface, ifp);
+    babel_interface_nfo *babel_ifp;
+
+    babel_ifp = babel_get_if_nfo(ifp);
+    assert (babel_ifp != NULL);
+
+    babel_ifp->channel = CHECK_FLAG(babel_ifp->flags, BABEL_IF_WIRED)? BABEL_IF_CHANNEL_NONINTERFERING: BABEL_IF_CHANNEL_INTERFERING;
```

Comment:
This formatting seems wrong to me, but @frrbot does not warn about that.

Reply:
Updated. Is there a guide to configure a linter for FRR?